static/bib/iasl.bib

```diff
@@ -1,3 +1,31 @@
+@article{matsubara-levorato-2020-icpr,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Neural Compression and Filtering for Edge-assisted Real-time Object Detection in Challenged Networks},
```

Comment:
The correction to the compound word capitalization in title case
[still][prev] seems to be missing:

```suggestion
  title = {Neural Compression and Filtering for Edge-Assisted Real-Time Object Detection in Challenged Networks},
```

[prev]: https://github.com/uci-iasl/website/pull/34#discussion_r471159058